docs/design.md

```diff
@@ -181,11 +189,35 @@ never be decreased. The core difference between the two isolation levels
 SI and SSI is that the former allows the transaction's candidate
 timestamp to increase and the latter does not.
 
-Timestamps are a combination of both a physical and a logical component
-to support monotonic increments without degenerate cases causing
-timestamps to diverge from wall clock time, following closely the
-[*Hybrid Logical Clock
-paper.*](http://www.cse.buffalo.edu/tech-reports/2014-04.pdf)
+**Hybrid Logical Clock**
+
+Each cockroach node maintains a hybrid logical clock (HLC) as discussed
+in the [*Hybrid Logical Clock
```

Comment:
Lose the emphasis in the link title. I think it was the conversion from doc to Markdown that did this.